rest-service/manager_rest/upload_manager.py

```diff
@@ -582,13 +583,14 @@ def _prepare_and_process_doc(self, data_id, blueprint_url,
             self.upload_archive_to_file_server(data_id)
 
         try:
-            temp_blueprint.upload_execution = rm.upload_blueprint(
+            temp_blueprint.upload_execution, messages = rm.upload_blueprint(
                 data_id,
                 application_file_name,
                 blueprint_url,
                 config.instance.file_server_root,   # for the import resolver
                 validate_only=True,
             )
```

Comment:
Unrelated to this set of commits (I think), but why does this one not have any rm.sm.update or similar?

Reply:
Just ORM Things™